webapp/src/shared/components/Snackbar/Snackbar.css

```diff
@@ -8,14 +10,14 @@
   border: none;
   border-radius: 0.75rem;
   min-height: 3rem;
-  min-width: 15rem;
+  min-width: 18rem;
   color: var(--color-light);
```

Comment:
thought: The white text on yellow/green background is a bit hard to read.
![yellow-white](https://user-images.githubusercontent.com/54190400/201090999-7acf4b35-cc13-4c88-81ab-a53a4617b969.png)
![green-white](https://user-images.githubusercontent.com/54190400/201091016-031588d3-f398-4c47-8b86-2f06c0cabf6c.png)

question: What do you think about changing this to `color: var(--color-background);`?
![yellow-black](https://user-images.githubusercontent.com/54190400/201091059-1e5ad8df-4df4-4089-800e-2209352b9fb6.png)

![green-black](https://user-images.githubusercontent.com/54190400/201091069-c3471ca5-37d7-40aa-b6a4-8590fddca421.png)


Source: https://accessibleweb.com/color-contrast-checker/

Reply:
Yes! nice idea :+1: